deploy-to-production.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
This is not for production deploy. That will be complete different one once we get everything sorted out with it.

Reply:
oh, this one for staging right?

Reply:
Yes.